pavelib/acceptance_test.py

```diff
@@ -13,22 +14,23 @@
 __test__ = False  # do not collect
 
 
-@task
 @needs(
     'pavelib.prereqs.install_prereqs',
     'pavelib.utils.test.utils.clean_reports_dir',
 )
 @cmdopts([
     ("system=", "s", "System to act on"),
-    ("default_store=", "m", "Default modulestore to use for course creation"),
+    ("default-store=", "m", "Default modulestore to use for course creation"),
     ("fasttest", "a", "Run without collectstatic"),
-    ("extra_args=", "e", "adds as extra args to the test command"),
     make_option("--verbose", action="store_const", const=2, dest="verbosity"),
     make_option("-q", "--quiet", action="store_const", const=0, dest="verbosity"),
     make_option("-v", "--verbosity", action="count", dest="verbosity"),
```

Comment:
Unfortunately, the interface we exposed is to use `-v` as a count, and the one nose exposes is to use `-v3` (with a number), so it's not quite a straight passthrough, from what I could tell.